### ISSUE TITLE
array - maxItems and minItems not displayed

### ISSUE THREAD

**Issue by kevinrenskers** [raml2html/default-theme#18]
See https://github.com/raml2html/raml2html/issues/383.

Example file: https://github.com/raml2html/raml2html/files/1741168/sampleRaml.zip

**Comment by kevinrenskers** [comment]
The template does have support for `minimum` and `maximum`. Did this change with RAML 1.0?

```
{%- if item.type == 'string' -%}
  {%- if item.minLength or item.minLength == 0 %} - minLength: {{ item.minLength }}{%- endif -%}
  {%- if item.maxLength or item.maxLength == 0 %} - maxLength: {{ item.maxLength }}{%- endif -%}
{%- else -%}
  {%- if item.minimum or item.minimum == 0 %} - minimum: {{ item.minimum }}{%- endif -%}
  {%- if item.maximum or item.maximum == 0 %} - maximum: {{ item.maximum }}{%- endif -%}
{%- endif -%}
```

**Comment by kevinrenskers** [comment]
Fixed in raml2html-default-theme version 2.7.1.

**Comment by muthulingapandian** [comment]
Thanks a lot Kevin, it worked.
